celsos_bookSearch/celsos_bookSearch.xcworkspace/xcuserdata/celsos.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -3,4 +3,22 @@
    uuid = "D13AE852-B91A-4C78-B566-A5856FA24CDF"
    type = "0"
    version = "2.0">
+   <Breakpoints>
```

Comment:
A .gitignore file should be added to the default repository tree to avoid those kind of info (xcuserdata) to be pushed to remote. You can find a default .gitignore online for iOS/swift projects.
This should be done to avoid personal debug information to be pushed to remote

Reply:
Sure, thanks for the hint

Reply:
Done